sample/src/main/java/com/farsitel/bazaar/sample/auth/MainActivity.kt

```diff
@@ -0,0 +1,76 @@
+package com.farsitel.bazaar.sample.auth
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.farsitel.bazaar.auth.BazaarHelper
+import com.farsitel.bazaar.auth.BazaarSignIn
+import com.farsitel.bazaar.auth.BazaarSignInClient
+import com.farsitel.bazaar.auth.callback.BazaarSignInCallback
+import com.farsitel.bazaar.auth.model.BazaarSignInAccount
+import com.farsitel.bazaar.auth.model.BazaarSignInOptions
+import com.farsitel.bazaar.auth.model.SignInOption
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+
+    private lateinit var loginButton: View
+    private lateinit var client: BazaarSignInClient
+    private val REQ_CODE = 123
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        checkUserAlreadySignedIn()
+
+        loginButton = findViewById(R.id.loginButton)
+    }
+
+    private fun checkUserAlreadySignedIn() {
+        val signInOption = BazaarSignInOptions.Builder(SignInOption.DEFAULT_SIGN_IN).build()
+        client = BazaarSignIn.getClient(
+            this,
+            signInOption
+        )
+
+        BazaarSignIn.getLastSignedInAccount(this,
+            this,
+            BazaarSignInCallback { account ->
+                updateUI(account)
+            })
```

Comment:
```kotlin
BazaarSignIn.getLastSignedInAccount(
    this,
    this,
    BazaarSignInCallback { account ->
        updateUI(account)
    }
)
```
Also, it would be nicer to use named arguments for this use-case.